src/stories/foundations/design_principles.stories.mdx

```diff
@@ -4,7 +4,19 @@ import { Meta } from '@storybook/addon-docs'
 
 # Design Principles
 
+🚧 **WIP!**
+
+## General principles
+
 - **Human Centered:** Specify context, be empathetic, test often, iterate
 - **Accessible:** All our product should be able to be used by as many people as possible. More on that in accessibility.
 - **Joyful:** Doing something good should feel good and be fun.
 - **Responsive:** Our products should work on all devices. Layouts should adapt and support.
+
+## Vision
+
+Our Vision is to create seamless, beautiful and professional design. Accessible for everyone. We want our users to find quickly what they need, have fun while using it and engage highly through them with others to create impact for good.
+
+## Mission:
+
+We want clarity, system and consistency in design. We want that once set design rules are used as strong guidelines and have leverage in discussions. Our design should be a facilitator for better communication and processes and not an obstacle in the way of innovation. The user / human should always be at the center of every decision.
```

Comment:
@inesczuia The last sentence only repeats what was already listed in the principles.

```suggestion
We want clarity, system and consistency in design. We want that once set design rules are used as strong guidelines and have leverage in discussions. Our design should be a facilitator for better communication and processes and not an obstacle in the way of innovation.
```